Randomization_try.py

```diff
@@ -4,161 +4,162 @@
 
 @author: Jacopo
 
-This script creates as many dataframes as participants, randomizing the position, color, and orientation of the items
-A controlled randomization is done on cue colors where matches with the color target are checked to be exaclty half of the
-lines checked; having three cues of the same color is nalso avoided."""
-
+This script creates as many dataframes as participants set in the init script,
+randomizing the position, color, and orientation of the items.
+"""
 
-import numpy as np, pandas as pd, math, os, time
+import math
+import os
+import time
+import numpy as np
+import pandas as pd
 
-wdir = os.getcwd()
-
-########################
-##     PARAMETERS     ##
-########################
+from Init_Cshape_rev import (trials_total, series, number_positions, colors,
```

Comment:
Probably messed up with the name of the files to import from, they should all be adapted (Init_Cshape_rev is Init_Cshape)